routers/postsRouter.js

```diff
@@ -5,8 +5,28 @@ const express = require('express')
 const router = express.Router();
 
 //Middleware 
+//...
+
 
 //Route Handlers
+//***POSTS ROUTE HANDLERS *** */
```

Comment:
Your future self and coworkers will thank you for labeling endpoints like this Christopher 👍